packages/grid_client/src/primitives/nodes.ts

```diff
@@ -274,7 +276,7 @@ class Nodes {
       .then(res => {
         const node: NodeCapacity = res;
         const ret: NodeResources = { cru: 0, mru: 0, hru: 0, sru: 0, ipv4u: 0 };
-
+        ret.cru = +node.capacity.total_resources.cru;
```

Comment:
Shouldn't the this be
```suggestion
        ret.cru = +node.capacity.total_resources.cru - +node.capacity.used_resources.cru;
```

Reply:
As per discussion with @AhmedHanafy725 we only need to check if the cru needed by the user is within the range of the total capacity on the node.